src/main/java/tr/com/srdc/cda2fhir/jolt/MedicationMap.java

```diff
@@ -0,0 +1,188 @@
+package tr.com.srdc.cda2fhir.jolt;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import tr.com.srdc.cda2fhir.conf.Config;
+import tr.com.srdc.cda2fhir.transform.util.impl.IdentifierMap;
+
+public class MedicationMap {
+
+	Map<String, MedicationContainer> ccMap = new HashMap<String, MedicationContainer>();
+
```

Comment:
Unless you intended to make this package wide, I would change it to private. If you omit the visibility Java default is package wide I believe.

Reply:
gotchya. Yeah I can make it private.